index.html

```diff
@@ -128,11 +129,52 @@ <h2 class="heading2 multy-task">Multi-Post Stories</h2>
                         Project</button>
                 </form>
             </div>
         </div>
+        <div class="popup">
+            <div class="popup-header">
+                <h1 class="popup-h1">Multi Post Stories</h1>
+                <h1 class="popup_h1_desktop">Keeping track of hundreds  of components website</h1>
+                <img class="popup-close" alt="close button" src="css/image/Bars/close.png">
+            </div>
+            <form class="popup-icons-container">
+                <button class="popup-icons">html</button>
+                <button class="popup-icons">Boptstrap</button>
+                <button class="popup-icons">Ruby on rails</button>
+            </form>
+            <div class="popup-section2">
+                <img class="popup-img" alt="multy post img" src="css/backgroundimg/multi_post_story.svg">
+                <div class="popup-discription-btn">
+                    <p class="popup-discription">Lorem Ipsum is simply dummy text of the printing and typesetting
+                        industry.
+                        Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown
+                        printer
+                        took a galley of type and scrambled it to make a type specimen book. It has survived not only
+                        five
+                        centuries, but also the leap into electronic typesetting, remaining essent</p>
+                    <p class="popup-discription-desktop">
+                        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been
+                        the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley
+                        of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing
+                        and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the
+                        1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map
+                        lapora verita.
+                    </p>
+                    <form class="popup-form">
+                        <button type="submit"
+                            class="animate-btn project-desktop-button button1 button1-text popup_btn">See
+                            Live <img class="see_live_icon" alt="see_live_image"
+                                src="css/Icons/see_live_Icon.png"></button>
+                        <button type="submit"
+                            class="animate-btn project-desktop-button button1 button1-text popup_btn">See
+                            Source <img class="see_live_icon" alt="see_live_image" src="css/Icons/git_hub.png"></button>
+                    </form>
+                </div>
+            </div>
+        </div>
```

Comment:
Your popup is not working properly, and popup is only implemented for one project. However, all the project cards should have a popup. Also the button for last project card is not working.